osprey-mock-service.js

```diff
@@ -133,17 +141,19 @@ function handler (method) {
       // Parse body.examples.
       if (Array.isArray(body.examples)) {
         example = []
-
         body.examples.forEach(function (ex) {
-          var obj = {}
-          obj[ex.name] = ex.structuredValue
-          example.push(obj)
+          example.push(ex.structuredValue)
         })
+        example = example[Math.floor(Math.random() * example.length)]
```

Comment:
can't we re-use `getSingleExample()` here? it is used only for headers examples for some reason but the (example(s)) logic is the same for both headers and bodies so I think we should re-use it.